example_test.go

```diff
@@ -87,3 +88,17 @@ func Example_structTags() {
 	// Output:
 	// db://
 }
+
+func Example_errorHandling() {
+	var c MyConfig
+	err := config.From("i_dont_exist.txt").FromEnv().To(&c)
```

Comment:
This is the really important part to look at: how exactly will the consumers use error handling?